samples/grid-pro/demo/validation/demo.js

```diff
@@ -0,0 +1,138 @@
+Grid.grid('container', {
+    dataTable: {
+        columns: {
+            productName: [
+                'Phone',
+                'Bread',
+                'T-Shirt',
```

Comment:
```suggestion
                'Shirt',
```
IMO the initial data should follow the rules